src/test/java/com/synopsys/integration/jenkins/ChangeSetFilterTest.java

```diff
@@ -0,0 +1,90 @@
+package com.synopsys.integration.jenkins;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import com.synopsys.integration.jenkins.extensions.JenkinsIntLogger;
+
+import hudson.scm.ChangeLogSet;
+import hudson.scm.EditType;
+
+public class ChangeSetFilterTest {
+
+    String validPath = "valid_path";
+    String invalidPath = "invalid_path";
+    String dummyPath = "dummy_path";
+
+    EditType editType = new EditType("MockEditTypeName", "MockEditTypeDescription");
+    JenkinsIntLogger MockJenkinsIntLogger = Mockito.mock(JenkinsIntLogger.class);
+    ChangeLogSet.AffectedFile mockAffectedFile = new MockAffectedFile(validPath, editType);
```

Comment:
Can Mockito.mock() and Mockito.when() be used here to avoid having to create an internal class?